### ISSUE TITLE
flit init does not respect existing pyproject.toml

### ISSUE THREAD

**Issue by WeirdKeksButtonK** [takluyver/flit#351]
Hi! I already have pyproject.toml (for Black) and there are no options to make `flit init` append content instead of rewriting the file. It is easy to bypass this limitation, but anyway, it may worth fixing.

**Comment by takluyver** [comment]
Yes, good point. That was originally written to create a flit-specific `flit.ini` file, and when I swapped over to `pyproject.toml`, basically no-one had the file already, so it was easy to create it from scratch. Now there are lots of projects using `pyproject.toml`.

[tomlkit](https://pypi.org/project/tomlkit/) may well be useful for this - it's meant to allow a program to modify TOML without losing comments & formatting.
